src/Ninja/G1CP/Content/Fixes/Session/fix235_DE_OrcDogMagBook.d

```diff
@@ -2,11 +2,9 @@
  * #235 Spelling - Magic book: "Verwandlung Orkhund" (DE)
  */
 func int G1CP_235_DE_OrcDogMagBook() {
-    var int idx; idx = G1CP_FindStringConstArrIdx("TXT_SPELLS", "Verwandlung Orc-Hund");
-    if (idx != -1) {
-        G1CP_SetStringConst("TXT_SPELLS", idx, "Verwandlung Orkhund");
-        return TRUE;
-    } else {
-        return FALSE;
-    };
-};
\ No newline at end of file
+    const string strConst = "TXT_SPELLS";
+    const string originalStr = "Verwandlung Orc-Hund";
+    const string correctedStr = "Verwandlung Orkhund";
+
+    return (G1CP_ConstReplaceStr(strConst, originalStr, correctedStr) > 0);
```

Comment:
This is good example of the comments in #361. I think these lines can be reduced to the following without loss of readability.
```D
return G1CP_ConstReplaceStr("TXT_SPELLS", "Verwandlung Orc-Hund", "Verwandlung Orkhund");
```